src/components/ContestResults.js

```diff
@@ -0,0 +1,69 @@
+import React, { useMemo } from "react";
+
+import LeaderboardTable from "./LeaderboardTable";
+
+function updateFindings(data, finding) {
+  data.allFindings += 1;
+  data.awardTotal += finding.awardUSD ?? 0;
```

Comment:
Never heard of the "Nullish coalescing operator" before! I learned something new!